app/forms/openid_connect_token_form.rb

```diff
@@ -122,6 +125,14 @@ def validate_aud_claim(payload)
                t('openid_connect.token.errors.invalid_aud', url: api_openid_connect_token_url))
   end
 
+  def validate_iat(payload)
```

Comment:
```suggestion
  # custom iat validation with some leeway because JWT gem is too strict
  def validate_iat(payload)
```

maybe drop a comment describing why we have this?